### PR TITLE
Introduce form components to create simple newsletter signups.

### PR DESCRIPTION
**Pull request by JPToroDev**
This PR introduces three form components—`Form`, `TextField`, and `SubscribeAction`—that allow users to craft basic email intake forms and create a foundation for future, full-fledged form creation.

Usage example:

```swift
Form {
    TextField(placeholder: "Email")
    Button("Submit")
        .type(.submit)
    } onSubmit: { id in
        SubscribeAction(.sendFox("myToken"), formID: id)
}
```

### CODE REVIEW

File: Sources/Ignite/Actions/SubscribeAction.swift [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    }
+
+    /// The endpoint URL for the form submission
+    var endpoint: String {
+        service.endpoint
+    }
+
+    /// Returns the JavaScript needed for form handling
+    public func compile() -> String {
+        """
```
Comment (twostraws) [inline Sources/Ignite/Actions/SubscribeAction.swift:65]: This whole PR is excellent – great job! This is the only part I'm not sure about: what JavaScript is required for the subscribe action?

Comment (JPToroDev) [reply Sources/Ignite/Actions/SubscribeAction.swift:65]: Thank you! Good catch—the JS just includes the form reset logic here. I'll update this with the subscription-handling logic that was omitted.

File: Sources/Ignite/Framework/ElementTypes/BlockHTML.swift [hunk 2]
```diff
@@ -24,11 +24,6 @@ extension BlockHTML {
     public func width(_ width: Int) -> Self {
         var copy = self
         copy.columnWidth = .count(width)
-
-        // Important: We need to transfer all our existing
-        // attributes over to the copy.
-        AttributeStore.default.merge(attributes, intoHTML: copy.id)
```
Comment (twostraws) [inline Sources/Ignite/Framework/ElementTypes/BlockHTML.swift:30]: I was reading through again and noticed this change – can this safely go? I seem to recall it causing problems in Ignite Samples.

Comment (JPToroDev) [reply Sources/Ignite/Framework/ElementTypes/BlockHTML.swift:30]: The copy should have the same id as the original, so this shouldn't be necessary.

Comment (twostraws) [reply Sources/Ignite/Framework/ElementTypes/BlockHTML.swift:30]: Great – thanks!

**Pull request merged**